cont3xt/vueapp/src/App.vue

```diff
@@ -113,81 +125,114 @@ export default {
+          break;
+        case 'KeyK':
+          // navigate up the indicator result tree
+          this.$store.commit('SET_RESULT_TREE_NAVIGATION_DIRECTION', 'up');
+          break;
+        case 'KeyH':
+          // navigate left in the indicator result tree
+          this.$store.commit('SET_RESULT_TREE_NAVIGATION_DIRECTION', 'left');
+          break;
+        case 'KeyL':
+          // navigate right in the indicator result tree
+          this.$store.commit('SET_RESULT_TREE_NAVIGATION_DIRECTION', 'right');
+          break;
+        }
+        return;
+      }
+
+      // shifted shortcuts
+      switch (e.code) {
+      case 'KeyQ':
         // focus on search expression input
         this.$store.commit('SET_FOCUS_SEARCH', true);
         break;
-      case 84: // t
+      case 'KeyT':
         // focus on start time input
         this.$store.commit('SET_FOCUS_START_DATE', true);
         break;
-      case 70: // f
+      case 'KeyF':
         // focus on time range selector
         this.$store.commit('SET_FOCUS_LINK_SEARCH', true);
         break;
-      case 86: // v
+      case 'KeyV':
         // focus on view dropdown selector
         this.$store.commit('SET_FOCUS_VIEW_SEARCH', true);
         break;
-      case 71: // g
+      case 'KeyG':
         // focus on tag input
         this.$store.commit('SET_FOCUS_TAG_INPUT', true);
         break;
-      case 69: // e
+      case 'KeyE':
         // toggle cache
         this.$store.commit('SET_TOGGLE_CACHE', true);
         break;
-      case 82: // r
+      case 'KeyR':
         // download report
         this.$store.commit('SET_DOWNLOAD_REPORT', true);
         break;
-      case 76: // l
+      case 'KeyL':
         // copy share link to clipboard
         this.$store.commit('SET_COPY_SHARE_LINK', true);
         break;
-      case 67: // c
+      case 'KeyC':
         // open cont3xt page if not on cont3xt page
         if (this.$route.name !== 'Cont3xt') {
           this.routeTo('/');
         }
         break;
-      case 65: // a
+      case 'KeyA':
         // open stats page if not on stats page
         if (this.$route.name !== 'Stats') {
           this.routeTo('/stats');
         }
         break;
-      case 89: // y
+      case 'KeyY':
         // open history page if not on history page
         if (this.$route.name !== 'History') {
           this.routeTo('/history');
         }
         break;
-      case 83: // s
+      case 'KeyS':
         // open settings page if not on settings page
         if (this.$route.name !== 'Settings') {
           this.routeTo('/settings');
         }
         break;
-      case 72: // h
+      case 'KeyH':
         // open help page if not on help page
         if (this.$route.name !== 'Help') {
           this.routeTo('/help');
         }
         break;
-      case 190: // . (seen as `>`, since shift is required)
+      case 'Period': // (seen as `>`, since shift is required)
```

Comment:
Thoughts about having < open the view panel to match?

Reply:
Sure! is adding that to this PR fine?

Reply:
yep